repeater/src/test/scala/com/snowplowanalytics/snowplow/storage/bigquery/repeater/SpecHelpers.scala

```diff
@@ -0,0 +1,214 @@
+      "v_etl":"spark-1.16.0-common-0.35.0",
+      "contexts_com_snowplowanalytics_snowplow_web_page_1_0_0":[
+      {
+        "id":"3e4a5deb-1504-4cb2-8751-9d358c632328"
+      }
+      ],
+      "contexts_org_w3_performance_timing_1_0_0":[
+      {
+        "chrome_first_paint":1458561380397,
+        "connect_end":1458561378602,
+        "connect_start":1458561378435,
+        "dom_complete":1458561383683,
+        "dom_content_loaded_event_end":1458561380395,
+        "dom_content_loaded_event_start":1458561380392,
+        "dom_interactive":1458561380392,
+        "dom_loading":1458561379095,
+        "domain_lookup_end":1458561378435,
+        "domain_lookup_start":1458561378211,
+        "fetch_start":1458561378195,
+        "load_event_end":1458561383737,
+        "load_event_start":1458561383683,
+        "ms_first_paint":null,
+        "navigation_start":1458561378194,
+        "proxy_end":null,
+        "proxy_start":null,
+        "redirect_end":0,
+        "redirect_start":0,
+        "request_end":null,
+        "request_start":1458561378602,
+        "response_end":1458561379080,
+        "response_start":1458561379079,
+        "secure_connection_start":0,
+        "unload_event_end":1458561379080,
+        "unload_event_start":1458561379080
+      }
+      ],
+      "contexts_com_snowplowanalytics_snowplow_ua_parser_context_1_0_0":[
+      {
+        "device_family":"Other",
+        "os_family":"Mac OS X",
+        "useragent_family":"Chrome",
+        "os_major":"10",
+        "os_minor":"10",
+        "os_patch":"5",
+        "os_patch_minor":null,
+        "os_version":"Mac OS X 10.10.5",
+        "useragent_major":"49",
+        "useragent_minor":"0",
+        "useragent_patch":"2623",
+        "useragent_version":"Chrome 49.0.2623"
+      }
+      ],
+      "contexts_org_ietf_http_cookie_1_0_0":[
+      {
+        "name":"sp",
+        "value":"05d09faa707a9ff092e49929c79630d1"
+      },
+      {
+        "name":"sp",
+        "value":"b73833bf7e3af5a6d5a02e79be96b064"
+      }
+      ]
+    }
+  """
+
+  def createContexts(flattenSchemaKey: String): Json = json"""{
+    $flattenSchemaKey:[
+      {
+        "name":"sp",
+        "value":"05d09faa707a9ff092e49929c79630d1"
+      },
+      {
+        "name":"sp",
+        "value":"b73833bf7e3af5a6d5a02e79be96b064"
+      }
+    ]
+  }
+  """
+
+  def createUnstructEvent(flattenSchemaKey: String): Json = json"""{
+    $flattenSchemaKey:
+      {
+        "chrome_first_paint":1458561380397,
+        "connect_end":1458561378602,
+        "connect_start":1458561378435
+      }
+    }
+    """
+
+  def minimalEventJson(id: UUID, collectorTstamp: Instant, vCollector: String, vTstamp: String): Json = json"""
+    {
+      "collector_tstamp" : $collectorTstamp,
+      "event_id":$id,
+      "v_collector":$vCollector,
+      "v_etl": $vTstamp
+    }
+    """
+
+  def minimalEvent(id: UUID, collectorTstamp: Instant, vCollector: String, vTstamp: String): Event =
```

Comment:
I wanted to have a constructor like this in SDK. Could you please create a ticket?